scripts/st2bootstrap-el8.sh

```diff
@@ -492,6 +496,13 @@ install_st2_dependencies() {
   if [[ -z "$is_epel_installed" ]]; then
```

Comment:
@armab I think that the situation is Rabbit will be in EPEL 8 it just isn't yet, it just isn't yet. Are you thinking just removing EPEL for now and putting it back in? 

I was deferring to the minimal changes (and keeping EPEL so that we are accommodating for the repo being present), and moving back to the package once it's released.

Reply:
I'm good with leaving it as is, considering it has a reason.

Can we document that we're waiting for RabbitMQ to appear in EPEL?
TODO Issue in this repo + Code comment linking to the issue would work best, just in case if we forget to revise it later.